tasks/push-rpm-manifests-to-pyxis/push-rpm-manifests-to-pyxis.yaml

```
@@ -59,6 +59,11 @@ spec:
           COMPONENT=$(jq -c --argjson i "$i" '.components[$i]' "${PYXIS_FILE}")
           IMAGEURL=$(jq -r '.containerImage' <<< "${COMPONENT}")
           NUM_PYXIS_IMAGES=$(jq '.pyxisImages | length' <<< "${COMPONENT}")
+          # Create a docker config file with just one entry for the image as partial oci matches
+          # result in authenticated errors
```

I would suggest saying something similar to what we say elsewhere (for oras):
```suggestion
          # cosign has very limited support for selecting the right auth entry,
          #  so create a custom auth file with just one entry
```

---

fixed